packages/shorebird_cli/test/src/commands/init_command_test.dart

```diff
@@ -215,8 +264,18 @@ Please make sure you are running "shorebird init" from the root of your Flutter
       );
       expect(
         File(p.join(tempDir.path, 'shorebird.yaml')).readAsStringSync(),
-        contains('app_id: $appId'),
+        contains('''
+app_id:
```

Comment:
We will need to update the rust library to handle this too I presume?

Reply:
No because I’m planning to add a post build step to adjust the bundled shorebird.yaml to include the app_id of the respective flavor so no changes should be required on the rust or backend.